src/main/java/org/nguh/nguhcraft/block/NguhBlocks.kt

```diff
@@ -252,6 +420,47 @@ object NguhBlocks {
         WROUGHT_IRON_BARS,
         GOLD_BARS,
         COMPRESSED_STONE,
+        CINNABAR,
+        CINNABAR_SLAB,
+        CINNABAR_STAIRS,
+        POLISHED_CINNABAR,
+        POLISHED_CINNABAR_SLAB,
+        POLISHED_CINNABAR_STAIRS,
+        POLISHED_CINNABAR_PRESSURE_PLATE,
+        POLISHED_CINNABAR_BUTTON,
```

Comment:
You forgot to remove the pressure plates and buttons here (and in `DROPS_SELF` and in the item groups below)